src/packages/documents/documents/tracked-reference/repository/document-tracked-reference.server.data.ts

```diff
@@ -38,19 +35,13 @@ export class UmbDocumentTrackedReferenceServerDataSource {
 	 * @return {*}
 	 * @memberof UmbDocumentTrackedReferenceServerDataSource
 	 */
-	async getTrackedReferenceDescendantsByParentId(
-		parentId: string,
-		skip = 0,
-		take = 20,
-		filterMustBeIsDependency = false,
-	) {
+	async getTrackedReferenceDescendantsByParentId(parentId: string, skip = 0, take = 20) {
```

Comment:
It doesn't look like this method is used. I think we should remove it for simplicity.